### PR TITLE
[ShopBundle][PayumBundle] Add payum authorize route

### PR DESCRIPTION
**Pull request by JaisDK**
| Q               | A
| --------------- | -----
| Branch?         | 1.1, 1.2
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by JaisDK** [comment]
@lchrusciel @pamil Are you the guys who look at PRs and can you take a look at this one?

File: src/Sylius/Bundle/PayumBundle/Action/AuthorizePaymentAction.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    {
+        $this->paymentDescriptionProvider = $paymentDescriptionProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @param Authorize $request
+     */
+    public function execute($request): void
```
[the remark this answers is not in the document]
Comment (JaisDK) [reply src/Sylius/Bundle/PayumBundle/Action/AuthorizePaymentAction.php]: spec added

**Comment by JaisDK** [comment]
@lchrusciel Is anything else needed to merge this=

**Comment by lchrusciel** [comment]
It seems to be fine for me. I would like to hear something from others too.

**Comment by Zales0123** [comment]
Hello @JaisDK, it's been a while, sorry for such a long feedback loop. The feature seems fine, however, I still feel a little bit uncomfortable without any Behat test for that :( I know, it's tricky and I have no idea how to do it properly, but it would be more than great - summon @pamil. The second thing, we should change a branch for sure (we don't develop `1.1` anymore), but should it be `master` or `1.2`? I still consider it as a feature, a little bit, as there was no support for *authorize* action and now there is - summon @pamil one more time :)
And we need to fix build asap to make it work :D Thank you contribution, I hope it will be merged in the very near future 🖖

**Comment by JaisDK** [comment]
Hi @Zales0123,

There was some support for authorize, the feature just wasn't completed, but I suppose an incomplete feature is the same as not having it.
Do you need me to change the base branch or are we waiting for @pamil to have a look?

**Comment by Zales0123** [comment]
cc @pamil, I would like to see your opinion :)

**Comment by loevgaard** [comment]
What's the status here, @pamil ? :)

**Pull request closed without merging**